data/achievements.cfg

```diff
@@ -1,5 +1,16 @@
 #textdomain wesnoth
 
+# macros for adding achievement icons
+#define ACHIEVEMENT_ICON_WBASE BASE_IMG ICON_IMG FRAME_IMG
+#arg FRAME_IMG
+misc/achievement-frame-mp.png#endarg
+{BASE_IMG}~SCALE(72,72)~BLIT({ICON_IMG}, 0, 0)~BLIT({FRAME_IMG}, 0, 0)#enddef
+
+#define ACHIEVEMENT_ICON ICON_IMG FRAME_IMG
+#arg FRAME_IMG
```

Comment:
If this is supposed to be an optional argument, it shouldn't be included in the `#define` line.

Reply:
This is still not resolved, and it also affects both macros.